javascript-if/main.js

```diff
@@ -0,0 +1,50 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else { return false; }
+}
+
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  } else { return false; }
+}
+
+function startsWithJ(string) {
+  if (string[0] === 'J') {
+    return true;
+  } else { return false; }
+}
+
+function isOldEnoughToDrink(person) {
+  if (person.age >= 21) {
+    return true;
+  } else { return false; }
+}
+
+function isOldEnoughToDrive(person) {
+  if (person.age >= 16) {
+    return true;
+  } else { return false; }
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return false;
+}
+
+function categorizeAcidity(pH) {
+  if (pH === 7) { return 'neutral'; } else if (pH > 7 && pH <= 14) { return 'base'; } else if (pH < 7 && pH >= 0) { return ' acid'; } else { return 'invalid pH level'; }
```

Comment:
You should always make each conditional block multiple lines, don't try to shorten it down to one line like this. Keeping code vertically oriented is **_much_** more preferable over horizontal scroll

Reply:
yeah I had them all in separate lines but when I saved the file, the editor just put them in one line